Suresh_TN/2_FibonacciSeries.py

```diff
@@ -0,0 +1,18 @@
+
+#WAP print Fibonacci series till 100 -
+
+def fibonacci(x):
+    if x<0:
+        return('Not Possible')
+    elif x==0:
+        return 0
+    elif x==1 or x==2:
+        return 1
+    else:
+        return(fibonacci(x-2)+fibonacci(x-1))
```

Comment:
Please write the code without recursion  because as we are not covered this topic